code4lib/2021.code4lib.org
# add hero image for 2021

To test:
- Confirm hero set as geometric image
- Adjust browser width and ensure hero image coverage through breakpoints

Closes #1
